azkaban-common/src/main/java/azkaban/imagemgmt/daos/ImageVersionDaoImpl.java

```diff
@@ -68,7 +68,7 @@
           + "values (?, ?, ?, ?, ?, ?, ?, ?, ?, ?)";
   private static final String SELECT_IMAGE_VERSION_BASE_QUERY =
       "select iv.id, iv.path, iv.description, "
-          + "iv.version, cast(replace(iv.version, '.', '') as unsigned integer) as int_version, "
```

Comment:
@abhisheknath2011 and I agreed on a phone call to let this PR go through since it adds a lot of testing. During merge he can add the unsigned and improve the test case to not trip on unsigned. The test case uses H2 db which does not allow unsigned as part of the query.